packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -665,5 +673,33 @@ contract Deploy is Deployer {
             console.log("ProxyAdmin ownership transferred to: %s", finalSystemOwner);
         }
     }
+
+    /// @notice Transfer ownership of the DisputeGameFactory contract to the final system owner
+    function transferDisputeGameFactoryOwnership() broadcast() public {
+        if (block.chainid == 900) {
+            DisputeGameFactory disputeGameFactory = DisputeGameFactory(mustGetAddress("DisputeGameFactoryProxy"));
+            address owner = disputeGameFactory.owner();
+            address finalSystemOwner = cfg.finalSystemOwner();
+            if (owner != finalSystemOwner) {
+                disputeGameFactory.transferOwnership(finalSystemOwner);
+                console.log("DisputeGameFactory ownership transferred to: %s", finalSystemOwner);
+            }
+        }
+    }
+
+    /// @notice Sets the implementation for the `FAULT` game type in the `DisputeGameFactory`
+    function setFaultGameImplementation() broadcast() public {
+        if (block.chainid == 900) {
+            DisputeGameFactory factory = DisputeGameFactory(mustGetAddress("DisputeGameFactoryProxy"));
+            Claim absolutePrestate = Claim.wrap(bytes32(cfg.faultGameAbsolutePrestate()));
+            IBigStepper faultVm = IBigStepper(new AlphabetVM(absolutePrestate));
```

Comment:
I think this is fine but to be idempotent we want to check that the impl isn't already set before setting it